analyzers/src/SonarAnalyzer.VisualBasic/SymbolicExecution/Roslyn/VisualBasicSyntaxClassifier.cs

```diff
@@ -37,6 +37,7 @@ node.Parent switch
             DoStatementSyntax doStatement => doStatement.WhileOrUntilClause,
             LoopStatementSyntax loopStatement => loopStatement.WhileOrUntilClause,
             ForBlockSyntax forBlock => forBlock.ForStatement,
+            ForEachStatementSyntax foreachStatement => foreachStatement.Expression,
```

Comment:
Misses https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntax.foreachvariablestatementsyntax?view=roslyn-dotnet-4.6.0

e.g `foreach(var (i, j) in new (int, int)[0])`

Should be `CommonForEachStatementSyntax` instead.